From d38397e62bce037acb788760f6befca7039a6614 Mon Sep 17 00:00:00 2001
From: Danny Kukawka <danny.kukawka@web.de>
Date: Fri, 7 Aug 2009 10:55:55 +0200
Subject: [PATCH 04/33] add keymap for a Samsung NC20

Added keymap for a Samsung NC20 as reported on HAL ML by
Dirk Thierbach <dthierbach@gmx.de>.
---
 fdi/information/10freedesktop/30-keymap-misc.fdi |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/fdi/information/10freedesktop/30-keymap-misc.fdi b/fdi/information/10freedesktop/30-keymap-misc.fdi
index c4e527e..2b5e45a 100644
--- a/fdi/information/10freedesktop/30-keymap-misc.fdi
+++ b/fdi/information/10freedesktop/30-keymap-misc.fdi
@@ -134,7 +134,7 @@
       </match>
 
       <match key="/org/freedesktop/Hal/devices/computer:system.hardware.vendor" prefix_ncase="samsung">
-	<match key="/org/freedesktop/Hal/devices/computer:system.hardware.product" contains_outof="NC10;SP55S;SQ45S70S;SX60P;SX30S;SX22S;R59P/R60P/R61P;Q210;Q310;X05;P560;R560">
+	<match key="/org/freedesktop/Hal/devices/computer:system.hardware.product" contains_outof="NC10;NC20;SP55S;SQ45S70S;SX60P;SX30S;SX22S;R59P/R60P/R61P;Q210;Q310;X05;P560;R560">
           <append key="input.keymap.data" type="strlist">e002:switchvideomode</append> <!-- Fn+F4 CRT/LCD (high keycode: "displaytoggle") -->
           <append key="input.keymap.data" type="strlist">e003:battery</append> <!-- Fn+F2 battery icon -->
           <append key="input.keymap.data" type="strlist">e004:prog1</append> <!-- Fn+F5 backlight on/off -->
-- 
1.7.1

